tailwind.config.js

```diff
@@ -9,11 +9,6 @@ module.exports = {
   presets: [],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {
-        backgroundImage: {
-         'hero-bg': "url('../assets/images/bghome.jpg')",
-        }
-      },
```

Comment:
This is not being used anywhere, and I dont think the compiler understands how to use the asset-pipeline anyway.